apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -13,28 +13,30 @@ import {
   shouldShowFile,
 } from '@codebridge/utils';
 import fileDownload from 'js-file-download';
-import React, {useContext, useMemo} from 'react';
+import React, {useMemo} from 'react';
 
 import codebridgeI18n from '@cdo/apps/codebridge/locale';
 import {START_SOURCES} from '@cdo/apps/lab2/constants';
 import {isReadOnlyWorkspace} from '@cdo/apps/lab2/lab2Redux';
 import {getAppOptionsEditBlocks} from '@cdo/apps/lab2/projects/utils';
 import {ProjectFileType} from '@cdo/apps/lab2/types';
 import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
-import {
-  DialogContext,
-  DialogType,
-} from '@cdo/apps/lab2/views/dialogs/DialogManager';
+import {useDialogControl, DialogType} from '@cdo/apps/lab2/views/dialogs';
 import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 
 import {FileBrowserHeaderPopUpButton} from './FileBrowserHeaderPopUpButton';
 import {
   downloadFileType,
   moveFilePromptType,
+  moveFileType,
```

Comment:
This is an example of something I want to get rid of by moving it to async - it needed restructuring to allow the prompted values to be accessed via a callback, and I opted to move them into separate functions instead of inside the callback. It's gonna be less work to unwind this later, it's just a little ugly atm.